src/js/modules/articles/components/ArticleForm.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from "react";
+import TitleInput from "./ArticleInputs/TitleInput";
+import ContentInput from "./ArticleInputs/ContentInput";
+import SectionInput from "./ArticleInputs/SectionInput";
+import ContributorsList from "./ContributorsList";
+import ContributorsInput from "./ArticleInputs/ContributorsInput";
+import injectSheet from "react-jss";
+import RaisedButton from "material-ui/RaisedButton";
+import FormErrorDialog from '../../forms/components/FormErrorDialog'
+
+const styles = {
+  button: {
+    maxWidth: "100px",
+    paddingTop: "5%"
+  }
+};
+
+class ArticleForm extends Component {
+
+  constructor(props) {
+    super(props);
+    const { title, content } = this.props;
+    this.state = {
+      title,
+      content,
+      section: 2
```

Comment:
Not sure what `section: 2` is supposed to do. If it's a default for drafts, maybe this should be made clearer with a constant or a comment.